content/docs/command-reference/plots/show.md

```diff
@@ -105,6 +105,36 @@ file:///Users/usr/src/plots/train.json.html
 
 > Note that only the last field name (`loss`) is used for the plot by default.
 
+<details>
+
+### Expand to see example using YAML metrics file
```

Comment:
Let's move this up right after the JSON code block and call it **Expand for YAML format**, please.

Reply:
Updated

Reply:
It's still not right after the JSON code block.